src/Compiler/CompilationResult.php

```diff
@@ -0,0 +1,305 @@
+<?php
+
+/**
+ * SCSSPHP
+ *
+ * @copyright 2012-2020 Leaf Corcoran
+ *
+ * @license http://opensource.org/licenses/MIT MIT
+ *
+ * @link http://scssphp.github.io/scssphp
+ */
+
+namespace ScssPhp\ScssPhp\Compiler;
+
+use ScssPhp\ScssPhp\Util;
+use ScssPhp\ScssPhp\Exception\CompilerException;
+
+/**
+ * Compiler environment
+ *
+ */
+class CompilationResult
+{
+    protected static $resolvedImport = [];
+
+    protected $isCached = false;
+
+    /**
+     * @var string
+     */
+    protected $css = '';
+
+    /**
+     * @var string
+     */
+    protected $sourceMap = '';
+
+    protected $sourceMapFile;
+    protected $sourceMapUrl;
+
+
+    /**
+     * All the effective parsedfiles
+     * @var array
+     */
+    protected $parsedFiles = [];
+
+    /**
+     * All the @import files and urls seen in the compilation process
+     * @var array
+     */
+    protected $includedFiles = [];
+
+    /**
+     * All the @import files resolved and imported (use to check the once condition)
+     * @var array
+     */
+    protected $importedFiles = [];
+
+
+    /**
+     * @param bool $isCached
+     */
+    public function setIsCached($isCached) {
+        $this->isCached = $isCached;
+    }
+
+    /**
+     * @return bool
+     */
+    public function getIsCached() {
+        return $this->isCached;
+    }
+
+    /**
+     * @param string $css
+     */
+    public function setCss($css)
+    {
+        $this->css = $css;
+    }
+
+    /**
+     * @param string $css
+     */
+    public function appendCss($css)
```

Comment:
AFAICT, this method is only used  to manage the prefix, where you even set the prefix as CSS and then append the actual CSS with this method. I'd rather not have this method here, and keep the concatenation in the compiler class.

Reply:
removed